doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,400 @@
+There is a common shared piece of setting up a dummy model and dataloader
+just for demonstration purposes.
+
+.. dropdown:: Wandb
+
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_model_dl.py
+        :language: python
+
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_wandb.py
+            :language: python
+            :start-after: __lightning_experiment_tracking_wandb_start__
+
+.. dropdown:: MLflow
 
-    def train_fn(config):
-        wandb = setup_wandb(config)
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_model_dl.py
+        :language: python
 
-        loss = optimize()
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_mlflow.py
+            :language: python
+            :start-after: __lightning_experiment_tracking_mlflow_start__
+            :end-before: __lightning_experiment_tracking_mlflow_end__
 
-        metrics = {"loss": loss}
+.. dropdown:: Comet
 
-        # No need to guard this behind the rank check anymore - Ray Train does
-        # this automatically in the returned `wandb` object.
-        wandb.log(metrics)
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_model_dl.py
+        :language: python
 
-        # Also report to Ray Train.
-        train.report(metrics)
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_comet.py
+            :language: python
+            :start-after: __lightning_experiment_tracking_comet_start__
 
+.. dropdown:: TensorBoard
+  
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_model_dl.py
+        :language: python
 
-Using Ray Train logger callbacks
---------------------------------
-Ray Train also exposes logging callbacks that automatically report results to
-experiment tracking services. This will use the results reported via the
-:func:`~ray.train.report` API.
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_tensorboard.py
+            :language: python
+            :start-after: __lightning_experiment_tracking_tensorboard_start__
+            :end-before: __lightning_experiment_tracking_tensorboard_end__
 
-Logger callbacks provide a simple tracking integration that don't require changes to your
-training code. If you need access to more fine grained APIs, use
-the :ref:`native integrations directly <train-experiment-tracking-native>`.
+Common Errors
+=============
 
-Example: Logging to MLflow and TensorBoard
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Missing Credentials
+-------------------
 
-**Step 1: Install the necessary packages**
+**I have already called `wandb login` cli, but still getting 
```

Comment:
nit: The formatting of this message and the one in the section below are slightly different. I would recommend making them consistent and putting the actual error message in a code block for readability purposes.